tests/functional/aws-node-sdk/test/mocha.opts

```diff
@@ -1,3 +1,4 @@
 --recursive
 --timeout 40000
 --ui tdd
+--bail
```

Comment:
If this option is the one that stops tests after one failure I would not include it for CI builds, its great locally but for CI we should show all errors.

Reply:
sorry, artifact from my ci testing, will remove

Reply:
removed